config/config.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright (C) InnoCraft Ltd - All rights reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains the property of InnoCraft Ltd.
+ * The intellectual and technical concepts contained herein are protected by trade secret or copyright law.
+ * Redistribution of this information or reproduction of this material is strictly forbidden
+ * unless prior written permission is obtained from InnoCraft Ltd.
+ *
+ * You shall use this code only in accordance with the license agreement obtained from InnoCraft Ltd.
+ *
+ * @link    https://www.innocraft.com/
+ * @license For license details see https://www.innocraft.com/license
+ */
+return array(
```

Comment:
It's not a big deal, but it's a little odd to use `array()` here and then `[]` for the rest of the lines.

Reply:
@snake14 updated to `[]`

Reply:
Thank you :+1: